src/repositories/projectUpdateRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { ProjectUpdate } from '../entities/project';
+
+export const findProjectUpdatesByProjectId = async (
+  projectId: number,
+): Promise<ProjectUpdate[] | []> => {
+  return ProjectUpdate.createQueryBuilder('projectUpdate')
+    .where('projectUpdate.projectId = :projectId', { projectId })
+    .andWhere('projectUpdate.isMain = true')
```

Comment:
@CarlosQ96  Why do we need this condition ? 
` .andWhere('projectUpdate.isMain = true')`

Reply:
Default projectUpdate is a hidden project update to add reactions to, only.

Reply:
I think Adminbro shouldn't really show it in the project view, doesn't add much.

Reply:
Pushed the change.

Reply:
Yeah now makes sense, thank you